src/content/blog/docker-providers.md

```diff
@@ -0,0 +1,58 @@
+| [Rancher Desktop](https://rancherdesktop.io/)                     |                                                 | New to DDEV                 | Yes | Yes |
+
+## What is a Docker Provider?
+
+DDEV depends on a Docker Provider, but what is it?
+
+Docker is the reason that DDEV can behave the same on every operating system, including Linux, WSL2, traditional Windows, and macOS (both Apple Silicon and Intel).
+
+All of the Docker Providers on every platform (except Linux) are actually wrappers on the open-source [Docker/Moby project](https://github.com/moby/moby), which is supported by [Docker, Inc](https://docker.com). Thank you! On Linux, the Docker setup runs natively, without any wrapper, which is why DDEV is so performant on Linux.
+
+## How can DDEV support all these providers?
+
+The idea of Docker is that everything happens inside a container, which you can think of as a little, separate computer. So the Docker images and containers used by DDEV are the same on every operating system, except that we build separate images for each [processor architecture](blog/arm64-apple-silicon-m1-ddev-local-what-does-it-all-mean/) (ARM64 - Apple Silicon, and AMD64 - Intel). So mostly all the work is done inside the container, and the containers/images behave the same. And DDEV’s main code is written in Golang, which is compiled natively into a single binary on every platform. Of course that sounds simple, but it’s not all that simple. Building all those images has quite a history, and of course Docker providers that ought to behave the same everywhere don’t actually, so there are workarounds in many places.
+
+## What are the Docker Providers on other Operating Systems?
+
+| OS | Provider | Open Source? | Free? | Comments                                                                           |
+| --- | --- | --- | --- |------------------------------------------------------------------------------------|
+| Traditional Windows | Docker Desktop | no | no | Can be buggy, can be slow, but pretty usable with Mutagen                          |
+| Windows WSL2 | docker-ce | yes | yes | Standard Linux Docker, performant, free, open source, well-maintained              |
+| Windows WSL2 | Docker Desktop | no | no | Can be buggy                                                                       |
+| Linux | docker-ce | yes | yes | Standard open-source Docker, well-maintained                                       |
+| Linux | Docker Desktop for Linux (DO NOT USE) | no | no | Non-functional for containers like DDEV that don’t use root user, poor performance |
+
+## What are DDEV Users Choosing Right Now?
+
+Traditional Windows users have only the Docker Desktop option, and nearly everyone on Linux uses docker-ce, but here are the macOS and WSL2 breakdowns:
+
+![Macos Docker Platform Usage](/img/blog/2023/11/macOS_Docker_Platform.png)
```

Comment:
The alt text here will describe the image for someone that may not be able to see it, so it might be nice to summarize what’s happening in the chart. (I’d use percentages but I don’t know what they are.)

```suggestion
![Donut chart depicting the last 7 days of macOS Docker Platform usage, led by almost three quarters Docker Desktop, then Colima, and a sliver of OrbStack](/img/blog/2023/11/macOS_Docker_Platform.png)
```